src/esp/io/JsonStlTypes.h

```diff
@@ -65,7 +65,7 @@ bool readMember(const JsonGenericValue& value,
       return false;
     }
     vec.reserve(arr.Size());
-    for (int i = 0; i < arr.Size(); i++) {
+    for (uint i = 0; i < arr.Size(); i++) {
```

Comment:
`size_t`? probably doesn't make much difference here but it makes tight loops faster on 64bit (as there's no need to generate extra instructions for extension/slicing of the 32bit type), so i'd say it's good to have it in "muscle memory" and use it always :)